packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -61,10 +61,76 @@ export const components: ThemeOptions["components"] = {
     },
   },
   MuiTypography: {
+    defaultProps: {
+      fontFamily:
+        "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
```

Comment:
If this is only for Storybook, we can yank it and rely on a hacky decorator.

Reply:
@edburyenegren-okta which PR has the font family fix?

Reply:
I wouldn't call it a fix 😅, but I've added this `style` wrapper to the decorator in my Radio PR: https://github.com/okta/odyssey/pull/1499/files#diff-dbaca548056fac468c040d8e2ec8eff229150099e56879178b3f1ea2ff1dfa80R12-R17